src/main/java/gregicadditions/machines/TileEntityFusionReactor.java

```diff
@@ -218,12 +219,12 @@ public void updateWorkable() {
 		@Override
 		protected Recipe findRecipe(long maxVoltage, IItemHandlerModifiable inputs, IMultipleTankHandler fluidInputs) {
 			Recipe recipe = super.findRecipe(maxVoltage, inputs, fluidInputs);
-			return (recipe != null && recipe.getIntegerProperty("eu_to_start") <= energyContainer.getEnergyCapacity()) ? recipe : null;
+			return (recipe != null && (long)recipe.getRecipePropertyStorage().getRecipePropertyValue(FusionEUToStartProperty.getInstance(), 0) <= energyContainer.getEnergyCapacity()) ? recipe : null;
```

Comment:
I'm a bit concerned about the differing instances of `long` and `int` for the EU To Start value. We should pick one and use it consistently.

Reply:
This is mostly due to a mismatch of energy values. The eu_to_start property is stored as an int in GTCE, although it might be viable to change this to a Long, which would have to wait on another release cycle, however, the energy container that it is being compared to holds energy as a Long

Reply:
Fair enough. Let's see if we can get that done in GTCE for the next cycle and we'll call this good enough for now.